### DISCUSSION TITLE
Методы и свойства для управления полетом игрока

### DISCUSSION THREAD

**Discussion by justqstn** [Sokol-Games-GmbH/PixelCombats.PublicApi#188] [Ideas]
свойства и методы для игрока
player.IsFlying {get;} - true если у игрока включен флай
player.FlyHeight {get; set;} - высота игрока с флаем
player.FlyOff(); player.FlyOn(); - выключить/включить флай
или добавить сеттер полю IsFlying, дабы изменять его

события для класса Players
Players.OnFlyState(player, state) - когда изменилось состояния полета у игрока

**Comment by Evgene82** [comment]
Не понял суть IsFlying, если можно просто прочитать через FlyEnable...

**Reply by justqstn** [reply]
тебе понимать не нужно - основная задумка остается такой же, а окончательно реализацию напишет костя

**Reply by Evgene82** [reply]
Добавлять новое свойство, когда есть идентичное?... А если изменить само свойство в Апи то будет несовместимость в режимах

**Reply by justqstn** [reply]
ну пусть и останется для обратной совместимости, нормальные люди будут писать по-другому. а может все будет реализовано в классе Build

**Comment by kkohno** [comment]
взяли на заметку
